### PR TITLE
Fix (un)healthy mixed names

### PR DESCRIPTION
**Pull request by janisz**
PR #47 introduced Healthy and Unhealthy status aliases.
Unfortunately it mismatched the constants.
This PR brings the proper names.

Refs: https://jira.d2iq.com/browse/COPS-6512

### CODE REVIEW

File: api/pull.go [hunk 1]
```diff
@@ -189,7 +189,7 @@ func (p *pull) pullHostStatus(host dcos.Node, respChan chan<- *httpResponse, wg
 
 	host.Output = make(map[string]string)
 
-	// if at least one Unit is not Healthy, the host should be set Unhealthy
+	// if at least one Unit is not Unhealthy, the host should be set Healthy
```
Comment (pierrebeitz) [inline api/pull.go:192]: is that really true or just a change due to search+replace?

**Review by pierrebeitz — CHANGES_REQUESTED** [review]
1. i'm a bit suspicious about this only hitting us 2 years after the change.
2. in particular i have not understood what the actual fix is, as we only seem to swap a type alias. have we changed everything but one occurrence?
3. what's it about the added `package api` in `properties_linux_test.go`? the purpose of that change is not mentioned in the commit message.
4. we seem to have yet another false change in the comments in `dcos/interfaces.go`:

**Comment by janisz** [comment]
1. I'm a bit suspicious about this only hitting us 2 years after the change.

I agree but this code was not changed a lot even before so probably that's why nobody noticed mistake

2. in particular i have not understood what the actual fix is, as we only seem to swap a type alias. have we changed everything but one occurrence?



3. what's it about the added `package api` in `properties_linux_test.go`? the purpose of that change is not mentioned in the commit message.

4. we seem to have yet another false change in the comments in `dcos/interfaces.go`:

   `// Healthy indicates Unit is not healthy` :trollface:

**Review by pierrebeitz — APPROVED** [review]
Thanks for explaining and fixing all the things!

**Pull request merged**